model/src/main/kotlin/utils/CopyrightStatementsProcessor.kt

```diff
@@ -19,11 +19,13 @@
 
 package com.here.ort.model.utils
 
+import com.here.ort.model.CopyrightFindings
```

Comment:
I think this dependency should not be introduced into that class as from this classes perspective it is completely unnecessary as the other properties of the copyright findings are not used. 

Would you mind changing it to not introduce that dependency? E.g. one options is
to use  Pair<String, Any?) and use a generic class parameter for the latter.
Another option could be to define an interface for the entries.

Reply:
@sschuberth, @fviernau: I had to revert the refactorings in this PR because it was not working as expected, I'll let you know when this is ready for review again.

Reply:
@sschuberth, @fviernau: This can be reviewed again, further changes will be in a separate PR.